src/pip/_internal/commands/debug.py

```diff
@@ -85,7 +81,6 @@ def get_module_from_module_name(module_name):
 
 def get_vendor_version_from_module(module_name):
     # type: (str) -> str
```

Comment:
There seems to be a possibility of returning `None`, so shouldn't the return type be `Optional[str]`?